### PR TITLE
Fix/broken links

### PR DESCRIPTION
**Pull request by Sednaoui**
Hi! Was learning more about yearn and fixed a few broken links while reading the docs, especially in faq

### CODE REVIEW

File: resources/faq.md [hunk 1]
```diff
@@ -48,19 +48,19 @@
 
 #### Who are Yearn's core developers?
```
Comment (lehnberg) [inline resources/faq.md:49]: /team/ has just been removed in a change, please see https://github.com/yearn/yearn-docs/pull/245

For now I propose we can proceed by removing the question of core developers from FAQ.
Multi-sig can be answered by linking to [YIP62](https://gov.yearn.finance/t/yip-62-change-two-multisig-signers/10758/)
 
What do you think?

**Review by lehnberg — CHANGES_REQUESTED** [review]
@Sednaoui thank you so much for taking the effort to fix these for us! 🙏 See my comment above, otherwise looks great.

File: resources/faq.md (hunk 1 above)
Comment (Sednaoui) [reply resources/faq.md:49]: sounds good! I will be making changes to governance links as well since they changed in #245

**Review by lehnberg — APPROVED** [review]
excellent, LGTM!

**Pull request merged**